src/content_scripts/chrome.js

```diff
@@ -0,0 +1,10 @@
+// communication type
+const UPDATE_CONTEXT_MENU = "updateContextMenu";
+
+document.addEventListener("selectionchange", () => {
+    const aselection = document.getSelection().toString();
```

Comment:
```suggestion
    const selection = document.getSelection().toString();
```

…and that of course then.